optical/converter/yolo.py

```diff
@@ -1,17 +1,136 @@
 """
-__author__: HashTagML
+__author__:HashTagML
 license: MIT
 Created: Wednesday, 31st March 2021
 """
 
-from .base import FormatSpec
-from typing import Union
+
 import os
+import imagesize
+from pathlib import Path
+import yaml
+import warnings
+from typing import Union
+import numpy as np
+import pandas as pd
+from .base import FormatSpec
+from .utils import exists, get_image_dir, get_annotation_dir
 
 
 class Yolo(FormatSpec):
     def __init__(self, root: Union[str, os.PathLike]):
-        pass
+        self.root = root
+        self.class_file = Path(os.path.join(self.root, "data.yaml"))
```

Comment:
I think it is better if a user provides `yaml` file name and path. If not warn the user that you are by default searching for `data.yaml`